easybuild/easyconfigs/g/GRIT/GRIT-2.0.5-intel-2016b-Python-2.7.12.eb

```diff
@@ -0,0 +1,41 @@
+easyblock = 'Bundle'
+
+name = 'GRIT'
+version = '2.0.5'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/nboley/grit'
+description = """GRIT - A tool for the integrative analysis of RNA-seq type assays"""
+
+toolchain = {'name': 'intel', 'version': '2016b'}
+
+# this is a bundle of Python packages
+exts_defaultclass = 'PythonPackage'
+
+dependencies = [
+    ('Python', '2.7.12'),
+]
+
+exts_list = [
+    ('networkx', '1.11', {
+        'source_urls': ['https://pypi.python.org/packages/source/n/networkx'],
+    }),
+    ('pysam', '0.9.1.4', {
+        'source_urls': ['https://pypi.python.org/packages/source/p/pysam/'],
+    }),
+    (name, version, {
+        'source_urls': ['https://github.com/nboley/grit/archive/'],
+        'source_tmpl': '%(version)s.tar.gz',
+    }),
+]
+
+modextrapaths = {'PYTHONPATH': ['lib/python%(pyshortver)s/site-packages']}
```

Comment:
open an issue for this in the easyblocks repo, I'll see if we can make that happen (or why it doesn't happen already)

I think the problem is that we don't want *every* extension to prepend to `$PYTHONPATH`, it just needs to be done once...